src/frontend/src/components/Posts.svelte

```diff
@@ -6,15 +6,17 @@
   import Alert from "./Alert.svelte";
   import { ROUTES } from "../router/routes";
   import MegaCoolUltraSpinner from "./MegaCoolUltraSpinner.svelte";
+  import { nip19 } from "nostr-tools";
 
   let profile = nostric_user.get_profile();
   let private_key = nostric_user.get_private_key();
   let message = null;
   let publishing = false;
-
+  let npub = nip19.npubEncode(profile.pk) 
+  
   const create_post = async () => {
     publishing = true;
-    let event = nostr_service.create_event(message);
+    let event = nostr_service.create_event(message, 1);
```

Comment:
v tej knižnici majú enum na tie Kinds, tu `nostr-tools/lib/event`. Importnúť to a všade to používať miesto čistého čísla. Takisto to definovať vo funkcii ako typ.